elasticsearch-evolution-core/src/main/java/com/senacor/elasticsearch/evolution/core/api/config/ElasticsearchEvolutionConfig.java

```diff
@@ -129,6 +135,7 @@ public ElasticsearchEvolutionConfig validate() throws IllegalStateException, Nul
             }
             requireNotBlank(historyIndex, "historyIndex must not be empty");
             requireCondition(historyMaxQuerySize, size -> size > 0, "historyMaxQuerySize value '%s' must be greater than 0", historyMaxQuerySize);
+            requireNotBlank(baselineVersion, "baselineVersion must not be empty");
```

Comment:
you should do a better validation here, e.g
```java
try {
  MigrationVersion.fromVersion(baselineVersion);
} catch (RuntimeException e) {
  throw new IllegalArgumentException("baselineVersion is invalid", e);
}
```

Reply:
Done